src/main/java/com/flagsmith/FlagsmithClient.java

```diff
@@ -398,7 +416,32 @@ public Builder setDefaultFlagPredicate(@NonNull Predicate<String> defaultFlagPre
      */
     public Builder setDefaultFlagValueFunction(
         @NonNull Function<String, String> defaultFlagValueFunction) {
-      this.client.defaultFlagValueFunc = defaultFlagValueFunction;
+      this.configuration.flagsmithFlagDefaults.setDefaultFlagValueFunc(defaultFlagValueFunction);
+      return this;
+    }
+
+    /**
+     * When a flag does not exist in Flagsmith or there is an error, the SDK will return an empty
+     * list of flags by default. For example:
+     * - if you call identifyUserWithTraits(...) and the call fails
+     * - this method will return an empty list of flags
+     *
+     * <p>If you would like the SDK to return a default list of flags with
+     * default values, you can set the default flag names with this method. For example:
+     * - if you se a default flag with the name "my-flag"
```

Comment:
As above, can we remove the leading hyphens here? There's also a minor typo on this line.

Reply:
Added proper javadoc list